versioning/versions/versions.go

```diff
@@ -22,6 +22,7 @@ type Package struct {
 
 type Version struct {
 	Dir          string
+	TemplateDir  string
```

Comment:
There are 2 thoughts:
- We can't put shared files in the root `template/` folder. This could be useful for some files that are version agnostic.
- We could make this an array to pick and choose which sub-dir to include.

Both of these can be potentially useful but I'm OK with implementing them in the future.

Reply:
We still can pick files from: templates folder (e.g. files to be shared between different versions) and a specific subfolder. It should be enough.

If we go with a array/list of subfolders then we will need to implement add'l logic to select specific files from a given subfolder. This will add add'l complexity - not sure if we need it.